src/plugins/tile_map/public/tile_map_type.js

```diff
@@ -38,15 +38,13 @@ import { createTileMapVisualization } from './tile_map_visualization';
 import { TileMapOptions } from './components/tile_map_options';
 import { supportsCssFilters } from './css_filters';
 import { truncatedColorSchemas } from '../../charts/public';
-import { getDeprecationMessage } from './get_deprecation_message';
 
 export function createTileMapTypeDefinition(dependencies) {
   const CoordinateMapsVisualization = createTileMapVisualization(dependencies);
   const { uiSettings, getServiceSettings } = dependencies;
 
   return {
     name: 'tile_map',
-    getInfoMessage: getDeprecationMessage,
```

Comment:
Is this prop referenced elsewhere?

Reply:
This is optional on the type visType. So removing it here satisfies the problem we are seeing.